search_that_hash/cracker/sth_mod/sth.py

```diff
@@ -0,0 +1,90 @@
+import json
+import requests
+from search_that_hash import printing
+
+
+class sth_api:
+    def __init__(self, config, hash_processes, sth_results):
+        self.hash_processes = hash_processes
+        self.config = config
+        self.sth_results = sth_results
+
+    def crack(config):
+        to_del = []
+        url = "https://av5b81zg3k.execute-api.us-east-2.amazonaws.com/prod/lookup"
+        payload = json.dumps({"Hash": list(config["hashes"].keys())})
+        headers = {"Content-Type": "application/json"}
+        try:
+            response = requests.request(
+                "GET", url, headers=headers, data=payload, timeout=config["timeout"]
+            )
+        except:  # If the user cannot connect
+            return False, config

```

Comment:
Use a more specific try except. What is the exact error that happens here? If you cannot think of it, and you cannot recreate it, it does not error so do use a try: except

Reply:
At least check the reponse like:

```
if response.status_code
```

try:  excepts are bad code, and it is super slow to deal with. If you cannot think of a specific error then you do not need them :)